storage/database/db_manager.go

```diff
@@ -272,9 +286,10 @@ var dbConfigRatio = [databaseEntryTypeSize]int{
 	5,  // BodyDB
 	5,  // ReceiptsDB
 	40, // StateTrieDB
-	40, // StateTrieMigrationDB
+	20, // StateTrieMigrationDB
```

Comment:
Is this value well defined? why does this value decrease by 20?

Reply:
I wasn't sure about the number. But, I think it is find with decreasing `StateTrimeMigrationDB` size since the data size will be reduced. I agreed that it took too much, so I think `3` is enough for snapshotDb. How does it sound?

Here is some numbers for database.
```
32%   415G	./body
14%   186G	./statetrie_migrated_79715161
0.6%  8.3G	./misc
0%    604K	./bridgeservice
12.7% 166G	./header
0.5%  63G	./txlookup
30%   392G	./receipts
1.3T	.
```

```
10%  283G	./body
1%   28G	./snapshot
0.2% 8.0G	./misc
0%   408K	./bridgeservice
5.7% 162G	./header
71%  2.0T	./statetrie_migrated_36097804
1.5% 44G	./txlookup
11%  305G	./receipts
2.8T	.
```

Currently, the snapshot takes about 30GB.
```
28G	./snapshot
```

Reply:
I'm not sure what number is appropriate for this. What about adding an issue for tuning this number?

Reply:
Alright. Let me add an issue for this and I will take a look later.

Reply:
Created an issue https://github.com/klaytn/klaytn/issues/1119